randomizer.py

```diff
@@ -1485,13 +1486,50 @@ def RandomizeDominion(setNames=None, options=None):
             counter += 1
 
```

Comment:
A general note: it seems to me the code was much cleaner when the bane was determined prior to the Way of the Mouse. Why was that flipped around?